package.json

```diff
@@ -16,7 +16,8 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "pree2e": "rm -rf .nyc_output coverage .next || true",
-    "check-coverage-limits": "nyc report --reporter=text-summary --check-coverage --statements 100"
+    "check-coverage-limits": "nyc report --reporter=text-summary --check-coverage --statements 100",
+    "lint": "eslint ./components/** -c .eslintrc.json --ext js,jsx"
```

Comment:
should we check providers, layouts and pages also?

Reply:
good point, yes lets add everything here, good for sanity check in general

Reply:
added!